pkgs/development/tools/extism-cli/default.nix

```
@@ -0,0 +1,43 @@
+{ lib, stdenvNoCC, fetchFromGitHub, python3, makeBinaryWrapper }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "extism-cli";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "extism";
+    repo = "cli";
+    rev = "97935786166e82154266b82410028482800e6061";
+    sha256 = "sha256-LRzXuZQt5h3exw43UXUwLVIhveYVFw/SQ2YtHI9ZnWc=";
+  };
+
+  buildInputs = [ python3 ];
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  configurePhase = false;
+  buildPhase = false;
```

```suggestion
  dontConfigure = true;
  dontBuild = true;
```
but unless they break things just leave them on

---

I just re-tested without either of these sets of options, without them I just see extra logs in the build that say:
```
configuring
no configure script, doing nothing
building
no Makefile, doing nothing
```
Just to clarify, is your preference to remove these 2 lines since it doesn't impact anything?

---

Yes, we don't need to disable things that work if left alone.